bundles/edu.kit.ipd.sdq.activextendannotations/src/edu/kit/ipd/sdq/activextendannotations/DelegateProcessorUtil.xtend

```diff
@@ -1,65 +0,0 @@
-package edu.kit.ipd.sdq.activextendannotations
-
-import org.eclipse.xtend.lib.macro.TransformationContext
-import org.eclipse.xtend.lib.macro.declaration.MutableClassDeclaration
-import org.eclipse.xtend.lib.macro.declaration.ResolvedMethod
-import org.eclipse.xtend.lib.macro.declaration.TypeReference
-import java.util.Map
-
-// TODO MK if active annotation for static delegation is added to Xtend then
-// merge all code of this class with org.eclipse.xtend.lib.annotations.DelegateProcessor$Util
```

Comment:
I don’t know anything about a `@DeleGate` annotation. Are there concrete plans for it? Wouldn’t *all* the code for `@StaticDelegate` be rendered obsolete once it’s released?